### PR TITLE
Custom attribute add wizard UX change

### PR DESCRIPTION
**Pull request by DimalChandrasiri**
### Purpose
This PR contains configuration changes required for custom attribute adding wizard.

### Related Issues
- Issue `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [x] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- Related PR `#1` or (None)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/claims/components/wizard/local-claim/basic-details-local-claims.tsx [hunk 1]
```diff
@@ -18,9 +18,13 @@
 
 import { TestableComponentInterface } from "@wso2is/core/models";
 import { Field, FormValue, Forms } from "@wso2is/forms";
+import { GenericIcon, Hint } from "@wso2is/react-components";
+import { InlineEditInput } from "@wso2is/react-components/src";
```
Comment (brionmario) [inline apps/console/src/features/claims/components/wizard/local-claim/basic-details-local-claims.tsx:22]: Shall we move this to `"@wso2is/react-components"`

File: modules/react-components/src/components/input/inline-edit-field.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+import { TestableComponentInterface } from "@wso2is/core/models";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { Grid, GridColumn, Icon, Input, InputOnChangeData, Popup } from "semantic-ui-react";
+
+export interface InlineEditInputPropsInterface extends TestableComponentInterface {
+    text: string;
+    textPrefix?: string;
+    textPostfix?: string;
+
```
Comment (brionmario) [inline modules/react-components/src/components/input/inline-edit-field.tsx:27]:
```suggestion
```

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/claims/components/wizard/local-claim/basic-details-local-claims.tsx (hunk 1 above)
Comment (DimalChandrasiri) [reply apps/console/src/features/claims/components/wizard/local-claim/basic-details-local-claims.tsx]: Fixed with 9ead70e

File: modules/react-components/src/components/input/inline-edit-field.tsx (hunk 2 above)
Comment (DimalChandrasiri) [reply modules/react-components/src/components/input/inline-edit-field.tsx]: Fixed with commit 9ead70e

**Pull request merged**